src/wp-admin/network/users.php

```diff
@@ -107,12 +118,27 @@
 
 							case 'notspam':
 								$user = get_userdata( $user_id );
+								if ( is_super_admin( $user->ID ) ) {
+									wp_die(
+										sprintf(
+											/* translators: %s: User login. */
+											__( 'The user %s is a network administrator and cannot be modified.' ),
```

Comment:
Sorry, I didn't realize this was an existing string used above. Reverting my suggestion with:

```suggestion
											__( 'Warning! User cannot be modified. The user %s is a network administrator.' ),
```